src/core/ext/filters/http/server/http_server_filter.h

```diff
@@ -26,10 +26,10 @@
 /* Processes metadata on the server side for HTTP2 transports */
 extern const grpc_channel_filter grpc_http_server_filter;
 
-namespace grpc_core {
-// Temporary code that allows servers to accept PUT requests. DO NOT USE WITHOUT
-// PERMISSION.
-void InternalOnlyDoNotUseUnlessYouHavePermissionFromGrpcTeamAllowBrokenPutRequests();
-}  // namespace grpc_core
+// A Temporary channel arg that allows servers to accept PUT requests. DO NOT
+// USE WITHOUT PERMISSION.
+#define GRPC_ARG_INTERNAL_ONLY_DO_NOT_USE_UNLESS_YOU_HAVE_PERMISSION_FROM_ALLOW_GRPC_TEAM_ALLOW_BROKEN_PUT_REQUESTS \
```

Comment:
Btw there's a typo in the channel arg - it has the word ALLOW twice.

Reply:
Done! Thanks!